### ISSUE TITLE
[VSCode][Accessibility]A11y_HDInsight Tools for VSCode_Azure HDInsights Tools(installation)_Keyboard :Keyboard focus is not visible for the controls present in side the Contributions tab

### ISSUE THREAD

**Issue by jingyanjingyan** [microsoft/vscode#76063]
- VSCode Version: 1.34.0​
- OS Version: Mac OS, windows 10

Environment Details:
https://marketplace.visualstudio.com/items?itemName=mshdinsight.azure-hdinsight​
Visual studio code version:1.34.0​
​
Repro Steps:​
1)Download and install HDInsights for VS code from the URL: https://marketplace.visualstudio.com/items?itemName=mshdinsight.azure-hdinsight​
2)Tab till the contributions tab and press enter on it in the installation page ​
3)Navigate to the controls present inside the contributions tab​
4)check whether the keyboard focus is visible on all the controls or not​
​
Actual Result:​
Keyboard focus is not visible for the controls present in side the Contributions Tab​
​
Expected Result:​
Keyboard focus should be visible on the controls present inside the Contributions Tab​
 ​
User Impact:​
Keyboard only users will not understand whether the focus landed on the control or not​
​
MAS Reference: ​
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}​

Does this issue occur when all extensions are disabled?: Yes/No

![Bug_278_Focus is not visible at dropdowns](https://user-images.githubusercontent.com/36756024/60069140-4b652180-9744-11e9-9a7c-daad5d65ce7f.png)

Reference bug from HDInsight team: https://dev.azure.com/bigdatabj/HDInsightVSCodeExtension/_workitems/edit/278
